bin/experiment-utils.sh

```diff
@@ -38,6 +38,7 @@ prepare_experiment_dir () {
 
     # Link in the vanilla vector file
     ln data/${map_name}/article_vectors.csv ${exp_dir}/vanilla_vectors.csv
+    ln data/${map_name}/article_labels.csv ${exp_dir}/chocolate_vectors.csv
```

Comment:
I don't think this is exactly right. I actually don't think the article_labels.csv needs to be copied at all. It can just be accessed from the map directory.